acceptance/main_test.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2015 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License. See the AUTHORS file
+// for names of contributors.
+//
+// Author: Peter Mattis (peter.mattis@gmail.com)
+
+// +build acceptance
+
+package acceptance
+
+import (
+	"flag"
+	"os"
+	"os/signal"
+	"testing"
+)
+
+var numNodes = flag.Int("n", 3, "the number of nodes to start (if not otherwise specified by a test)")
+var stopper = make(chan struct{})
+
+func TestMain(m *testing.M) {
+	go func() {
+		sig := make(chan os.Signal, 1)
+		signal.Notify(sig, os.Interrupt)
+		<-sig
+		close(stopper)
```

Comment:
This will cause the current test to exit early, but then any subsequent tests will still run. We probably want to exit the whole process once the current test has cleaned up.